docs/modules/migrate/pages/which-tool-to-use.adoc

```diff
@@ -0,0 +1,29 @@
+= Which Tool Should I Use?
+:description: This flow chart will help you determine which upgrade or migration tool you should use for your Hazelcast cluster.
+
+{description} 
+
+Hazelcast provides several different options for upgrading your Hazelcast cluster and/or migrating your data from one cluster to another. 
+
+* The Data Migration Tool (DMT): Migrate data from your Open Source or Enterprise clusters running Hazelcast 4.0 or later, to either an Enterprise cluster or a cluster on Hazelcast Viridian. xref:data-migration-tool.adoc[]
+* Data Synchronization using Active/Active WAN Replication: Used to migrate data and active clients from one Enterprise cluster to another Enterprise cluster. xref:wan:wan.adoc[]
+* 3.12 Migration jar: A specialized tool for migrating data only from a 3.12 cluster to the latest version of Hazelcast. xref:migration-tool.adoc[]
+* Rolling Upgrade: Upgrades your existing 4.0 or later Enterprise cluster to the latest version of Hazelcast with no service interruption. xref:maintain-cluster:rolling-upgrades.adoc[]
+
+The method you use will depend on your version of Hazelcast and your operational conditions. This flow chart will guide you through the process of determining which method is available for your environment. Note that some options, such as data migration from one open source cluster to another open source cluster, are not available. 
+
+image::ROOT:migration-flowchart.png[migration flow chart, 1024]
```

Comment:
I think our documentation does not allow showing original images by clicking on it, it does not look nice currently. It looks like stretched vertically and a bit small. 
 
<img width="1728" alt="image" src="https://github.com/hazelcast/hz-docs/assets/32355782/0107ece3-c73c-4497-badc-aff5afafec5c">

Reply:
It is elongated - it's portrait-oriented. I will make it wider than the 1024 it's currently set to.

Reply:
It seems the same to me, is it expected? This is the link: https://deploy-preview-896--hardcore-allen-f5257d.netlify.app/hazelcast/5.4-snapshot/migrate/which-tool-to-use

Reply:
Look at the updated file; I pushed a change to 1920 width. Should be legible now.